### DISCUSSION TITLE
How to use README.md in the docs?

### DISCUSSION THREAD

**Discussion by felipecrs** [facebook/docusaurus#3462] [General]
I failed miserably trying to find information about it, I can't believe I'm the only one seeking for such a thing.

I want to use my project's `README.md` as one of the docs in docusaurus v2, specifically my entry-point documentation at `/`.

Questions:

1. How to point a single file out of the `docs` folder to be included in the docs?
2. How to set `README.md` as the entry point without manually writing the Front Matter on it (since I don't want it to be rendered in the repository's home on GitHub). Remember that `homePageId` was deprecated in docusaurus config.
3. How to set `hide_title: true` without manually writing the Front Matter on it, for the same reason of point 2.

**Comment by slorber** [comment]
You are likely not the only one. I've seem md-based docs framework supporting this, and also other systems not supporting this (including Github afaik, you can't import docs/README.md from README.md).

Shared a workaround on https://github.com/facebook/docusaurus/issues/3475#issuecomment-700603683
You can import one MDX file into the other to share bits of markdown content., so you could have a docs/README.md that has some frontmatter like `hide_title: true`, and would just import docs/README_CONTENT.md for example.

We'll try to support better these workflows. Ideally I think it would be nice to be able to totally reuse the github docs so that they are browsable from both Github or Docusaurus site, but it's not so simple due to different markdown parsers and ways docs are linked. We must find the common denominator of features supported by both systems (currently not well defined) and only use those.

**Reply by felipecrs** [reply]
Actually, I'm the same person who created the issue and proposed the workaround, lol.
